pages/apis/rest_api/limits.md

```diff
@@ -0,0 +1,34 @@
+
+# REST API rate limits
+To ensure stability and prevent excessive or abusive calls to the server, Buildkite imposes a limit on the number of REST API requests that can be made within a minute. These limits apply to the Pipelines REST API as well as the Analytics REST API
+
+## Rate limits
+Buildkite imposes a rate limit of 200 requests per minute for each organization. This is the cumulative limit of all API requests made by users in an organization.
+
+## Checking rate limit details
+The rate limit status is available in the following response headers of each API call.
+
+- `RateLimit-Remaining` - The remaining requests that can be made within the current time window
+- `RateLimit-Limit` - The current rate limit imposed on your organization.
+- `RateLimit-Reset` - The number of seconds remaining until a new time window is started and limits are reset.
+
+For example
+
+```js
+RateLimit-Remaining: 180
+RateLimit-Limit: 200
+RateLimit-Reset: 42
+```
+
+## Exceeding the rate limit
+Once the rate limit is exceeded, subsequent API requests will return a 429 HTTP status code, and the `RateLimit-Remaining` header will be 0. You should not make any further requests until the `RateLimit-Reset` specifies a new availability window.
```

Comment:
suggestion: You will also receive a `"RateLimit-Warning"` header informing you that the limit has been exceeded?

Reply:
Once we start rate limiting, I think that can be removed - The warning is only useful if we're not actually enforcing it. The 429 error itself would indicate rate limiting and the warning isn't our standard rate limiting header